utils/node_local_test.py

```diff
@@ -583,31 +581,30 @@ def start(self):
                              '--undef-value-errors=no']
             self._io_server_dir = tempfile.TemporaryDirectory(prefix='dnt_io_')
 
-            fd = open(os.path.join(self._io_server_dir.name,
-                                   'daos_engine'), 'w')
+            fd = open(join(self._io_server_dir.name, 'daos_engine'), 'w')
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations